pom.xml

```diff
@@ -325,6 +325,24 @@
             <scope>runtime</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.mortbay.jetty</groupId>
```

Comment:
hadoop-azure depends on hadoop-commons which depends on jetty-util. And hadoop-azure needs jetty-util, `AzureNativeFileSystemStore` uses it.

```
[INFO] org.apache.hadoop:hadoop-azure:jar:2.7.2
[INFO] +- org.apache.hadoop:hadoop-common:jar:2.7.2:compile
[INFO] |  +- org.apache.hadoop:hadoop-annotations:jar:2.7.2:compile
[INFO] |  |  \- jdk.tools:jdk.tools:jar:1.8:system
[INFO] |  +- commons-cli:commons-cli:jar:1.2:compile
[INFO] |  +- org.apache.commons:commons-math3:jar:3.1.1:compile
[INFO] |  +- xmlenc:xmlenc:jar:0.52:compile
[INFO] |  +- commons-httpclient:commons-httpclient:jar:3.1:compile
[INFO] |  +- commons-codec:commons-codec:jar:1.4:compile
[INFO] |  +- commons-net:commons-net:jar:3.1:compile
[INFO] |  +- commons-collections:commons-collections:jar:3.2.2:compile
[INFO] |  +- javax.servlet:servlet-api:jar:2.5:compile
[INFO] |  +- org.mortbay.jetty:jetty:jar:6.1.26:compile
[INFO] |  +- org.mortbay.jetty:jetty-util:jar:6.1.26:compile
[INFO] |  +- javax.servlet.jsp:jsp-api:jar:2.1:runtime
```
We can either remove the exclusion from hadoop-common or add it as top level dependency.